### DISCUSSION TITLE
Images with DynamicTable

### DISCUSSION THREAD

**Discussion by riicardojs** [pdfme/pdfme#519] [Ideas]
Hello,

I use pdfme to create the templates for my platform, and I've reached a point where I need to create a dynamic table that includes images or some other plugin that allows me to create a structure with text, images, and assets. This block should repeat in the same format as a table.

Below is an example of the information I'd like to repeat. Do you have any ideas on how I could solve this problem? It would be an array of information.

![image](https://github.com/user-attachments/assets/20642e90-7605-4d04-8ebb-1da6d255b6ae)

![image](https://github.com/user-attachments/assets/9ca05c93-b85b-4453-a75f-e171603be592)

**Comment by hand-dot** [comment]
Hey @riicardojs 
Currently, this is not possible, but I believe the following issue is related:
https://github.com/pdfme/pdfme/issues/484

**Comment by jsgosh** [comment]
Related issue: #671
